tools/lou_maketable

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+CURDIR=$(cd $(dirname "$0") && pwd)
+export LD_LIBRARY_PATH=$CURDIR/../liblouis/.libs
```

Comment:
This is really fishy. I presume it is needed to support the use case of running the tools directly in the `src` without installing them. I personally would simply not support that use case.

Reply:
Hmm, why do you find it fishy? Autotools does exactly the same.

I think the use case is important. The reason for supporting it is exactly the same as for Autotools supporting it for the other command line programs. Personally I never run `make install`, always use all of the tools right from within the source directory. Not being able to do that would be a big step backwards.

Reply:
OK, fine. Accessing `.libs` just feels like assuming things that ought to be hidden by libtool, IMHO.

Reply:
OK if your point is that this thing will break in the event of libtool suddenly changing the file paths, you're right. But I can live with that risk :)